lib/vmpooler.rb

```diff
@@ -18,4 +18,47 @@ module Vmpooler
       require File.expand_path(File.join(File.dirname(__FILE__), 'vmpooler', lib))
     end
   end
+
+  def self.config
+    # Load the configuration file
+    config_file = File.expand_path('vmpooler.yaml')
+    $config = YAML.load_file(config_file)
```

Comment:
I did not look at changing the variable's type when relocating the logic. Should this be changed or can it wait?